dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -567,6 +574,8 @@ public void changeDockerfiles(Namespace ns,
                     forkedRepo,
                     pullRequestInfo,
                     rateLimiter);
+        } else {
+            log.info("No files changed in repo {}. Skipping PR creation attempt.", parentName);
```

Comment:
This contains the org name as well rightr?

Reply:
yes